packages/common/src/ceramic-api.ts

```diff
@@ -8,6 +8,30 @@ import type { IndexApi } from './index-api.js'
 import { NodeStatusResponse } from './node-status-interface.js'
 import type { AnchorStatus } from '@ceramicnetwork/common'
 
+/**
+ * Field definition for index
+ */
+export interface Field {
```

Comment:
```suggestion
export type Field = {
```
For consistency with the other types defined below